content/docs/buildpack-author-guide/caching-strategies.md

```diff
@@ -0,0 +1,65 @@
+
+Setting `Cache = true` allows additional fine-grained control over caching.  The `Cache = true` flag caches a layer and allows a buildpack to make _content_ level decisions about the validity of the cache (as opposed to using the less granular metadata).  As an example, suppose a layer where `Launch = true` installs a `jq` binary with version `1.5` and sets `version=1.5` in the layer metadata.  By default, this layer will not be re-used from the registry when a buildpack requests `jq` with `version=1.6` to be installed.  However, setting `Cache = true` makes a previously built layer available during the build.  A buildpack could then prefer to implement logic to restore `jq` with `version=1.5` instead of performing a potentially expensive download of `jq` with `version=1.6`.  The `Cache = true` setting allows for cache validation decisions to be made at a level of granularity that is much finer grained than layer metadata.
+
+Setting `Cache = false, Build = false, Launch = true` is the most common configuration.  If `Cache = false, Build = false, Launch = true` is not appropriate for your layer, then `Cache = true, Build = true, Launch = true` should be the next combination to evaluate:
+
+* When `Cache = true, Build = true, Launch = true`, explicitly setting `Build = true` makes the layer available, to subsequent buildpacks, during the build phase.  As `Cache = true` the layer is restored from local cache before proceeding to the build phase.  For example, we might provide a Python distribution in a cached, build and launch layer.  The build phase could verify that the restored cached version of our Python distribution contains Python 3.10 but disregard the patch number of the Python interpreter.
+* `Cache = true, Build = true, Launch = true` is an appropriate setting for a layer providing a distribution or runtime such as a Python interpreter or NodeJS runtime.
+
+Other common configurations include
+
+* `Cache = true, Build = false, Launch = true` Allows the same caching behavior as `Cache = true, Build = true, Launch = true`, but the layer is not available to subsequent buildpacks.  For example, the build phase can restore a Python distribution disregarding the patch number of the `major.minor.patch` number stored in the metadata.  As `Build = false` the python interpreter is unavailable to subsequent Buildpacks.
+* `Cache = true, Build = true, Launch = false` This configuration is useful where a build time dependency is provided.  For example, we might provide a JDK as a cached build layer that is not added as a launch layer.  Instead we might provide a JRE as a launch layer in the application image.
```

Comment:
>For example, we might provide a JDK as a cached build layer that is not added as a launch layer.  Instead we might provide a JRE as a launch layer in the application image.

For example, a JDK could be provided as a cached build layer that is not added as a launch layer.  Instead, it might provide a JRE as a launch layer in the application image.

Reply:
:)